jobsdb/jobsdb.go

```diff
@@ -1066,14 +1066,12 @@ func (jd *HandleT) refreshDSRangeList(l lock.LockToken) {
 		jd.assert(ds.Index != "", "ds.Index is empty")
 		sqlStatement := fmt.Sprintf(`SELECT MIN(job_id), MAX(job_id) FROM %q`, ds.JobTable)
 		// Note: Using Query instead of QueryRow, because the sqlmock library doesn't have support for QueryRow
-		rows, err := jd.dbHandle.Query(sqlStatement)
+		row := jd.dbHandle.QueryRow(sqlStatement)
+		jd.assertError(row.Err())
+
+		err := row.Scan(&minID, &maxID)
```

Comment:
Using `QueryRow` since we only wanted to read a single row 

```suggestion
		row := jd.dbHandle.QueryRow(sqlStatement)

		err := row.Scan(&minID, &maxID)
```